pkg/upgrade/main.go

```diff
@@ -0,0 +1,36 @@
+package upgrade
+
+import (
+	"sort"
+
+	"github.com/Masterminds/semver"
+)
+
+func init() {
+	parseSemVer()
+}
+
+func parseSemVer() {
+	// ignore errors, we shouldn't have semantic version parsing errors at runtime
+	semanticVersions, _ = versions(upgrades)
```

Comment:
The error needs to be reported, as otherwise would not be detected in development or testing. Possibly should even cause a panic, so would definitely be picked up immediately in dev/test environment? It should never happen.

Reply:
panic is a good option  IMHO.